adoc/admin-security-rbac.adoc

```diff
@@ -134,10 +139,13 @@ Besides the LDAP connector you can also set up other connectors.
 For additional connectors, refer to the available connector configurations in the Dex repository:
 https://github.com/dexidp/dex/tree/v2.16.0/Documentation/connectors.
 +
-. Update Dex deployment:
+. Save and exit Dex ConfigMap by typing `:wq` in the terminal.
++
+. Restart Dex and Gangway by running:
 +
 ----
-kubectl replace --force -f <cluster-name>/addons/dex/dex.yaml
+kubectl --namespace=kube-system delete pod -l app=oidc-dex
+kubectl --namespace=kube-system delete pod -l app=oidc-gangway
```

Comment:
I don't think we need to restart gangway pods since this is for fix dex configuration.